physionet-django/user/forms.py

```diff
@@ -429,7 +432,10 @@ class Meta:
             'reference_email', 'reference_title')
         help_texts = {
             'reference_category': """Your reference's relationship to you. If 
-                you are a student or postdoc, this must be your supervisor.""",
+                you are a student or postdoc, this must be your supervisor. 
+                Otherwise, you may list a colleague. Do not list yourself 
+                or another student as reference. Remind your reference to respond 
+                promptly as long response times will prevent application approval.""",
```

Comment:
Minor thing, but I think this needs a comma after promptly - > "respond promptly as long response..". While you are here, perhaps also tweak "will prevent application approval." to "will prevent approval of your application."?